docker-compose.yaml

```diff
@@ -5,7 +5,7 @@ services:
     image: archwaynetwork/archwayd:latest
     build:
       context: .
-    command: ["sh", "/opt/localnet.sh"]
+    entrypoint: ["sh", "/opt/localnet.sh"]
```

Comment:
Why this change is in this PR?

Reply:
ill remove this to a separate chore issue.